leases/app/controllers/MediaLeaseController.scala

```diff
@@ -38,13 +37,14 @@ object MediaLeaseController extends Controller
 
   val notFound = respondNotFound("MediaLease not found")
 
-  val appIndex = AppIndex(
-    "media-leases",
-    "Media leases service",
-    Map(),
-    List(Link("by-media-id", s"$rootUri/leases/media/{id}"))
-  )
-  def index = Authenticated { _ => respond(appIndex) }
+
+  val indexResponse = {
+    val appIndex = AppIndex("media-leases", "Media leases service", Map())
+    val indexLinks =  List(Link("by-media-id", s"$rootUri/leases/media/{id}"))
```

Comment:
Hey guys :smile: 

It could work as a Link, but making it an Action might be nicer. I see the problem with URI, technically the field should probably allow URI templates (which is a superset of URIs), so might have to be changed back to String indeed. Note that the Actions are inspired by the Swagger spec, which [allows](http://swagger.io/specification/#pathsObject) URI templates in a similar context.